src/utils/shell.mjs

```diff
@@ -48,7 +48,11 @@ export const runCommand = (command, env = {}, spinner = null) => {
     preferLocal: true,
     // we use reject=false to avoid rejecting synchronously when the command doesn't exist
     reject: false,
-    env,
+    env: {
+      // we want always colorful terminal outputs
+      FORCE_COLOR: 'true',
```

Comment:
This makes our framework dev commands with colorful output!

### old
![CleanShot 2023-07-10 at 09 51 43](https://github.com/netlify/cli/assets/11156362/c72ba6de-ab14-4906-b064-a04f0882711a)

### new 


![CleanShot 2023-07-10 at 09 48 43](https://github.com/netlify/cli/assets/11156362/6d8de0a0-3682-4972-a47b-9cf72c9c95f5)